storybook/header-preset/preset.ts

```diff
@@ -1,11 +1,11 @@
 import path from "path"
 import { NormalModuleReplacementPlugin } from "webpack"
-const webpackConfig = require("../webpack.config")
+const webpackConfig = require("../rules")
 
 module.exports = {
-  managerWebpack: async config => {
+  managerWebpack: (config, { configType }) => {
```

Comment:
Storybook gives us an env variable depending on whether you're running build-storybook or start-storybook, so setting our own variables isn't needed IMO.